source/clear-linux/clear-linux.rst

```diff
@@ -3,16 +3,23 @@
 Clear Linux\* Project for Intel® Architecture
 #############################################
 
-Welcome to the |CLOSIA| documentation pages. Our documentation is split into
-five sections: Get started, Concepts, Guides, Tutorials, and Reference.
-Under :ref:`get-started`, you can find information about installing |CL| on
-bare metal, in a virtual environment, or as a live image on a USB stick. Under
-the :ref:`concepts` section, you can find detailed technical information about
-the |CL| features. The :ref:`guides` section contains step-by-step
-instructions to complete common tasks. The :ref:`tutorials` section contains
-step-by-step instructions to complete the installation and configuration of
-tools needed for a specific use case. Lastly, the :ref:`reference` section
-contains information providing additional context or details.
+Welcome to the |CLOSIA| documentation pages. Our documentation is divided 
+into five sections:
+
+* :ref:`get-started` Find information about installing |CL| on
+  bare metal, in a virtual environment, or as a live image on a USB stick. 
+
+* :ref:`concepts` Find detailed technical information about 
+  |CL| features and what differentiates it from other |CL| distros. 
+
+* :ref:`guides` Find step-by-step instructions on how to complete 
+  common tasks that are critical to understanding |CL| and its features. 
```

Comment:
Just a quibble, but I wonder if these are critical to understand or maybe it would be better to say critical to being productive.